docs/docs/contribute/code/building-clamor.md

```diff
@@ -0,0 +1,57 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# Building Clamor
+
+*Before you start, ensure you've [set up your development environment](../getting-started/#development-environment).*
+
+This guide will outline the process to build [Clamor](https://github.com/fragcolor-xyz/clamor) from the sources.
+
+## Windows
+
+!!! note
+    Fragcolor doesn't yet officially support building and developing Clamor on the Windows platform. Please use Linux or Mac for Clamor. You may use the Linux/Mac [instructions](#linuxmac) on Windows with [WSL](https://docs.microsoft.com/en-us/windows/wsl/) but your mileage may vary and results are not guaranteed. Clamor is based on [Substrate](https://substrate.io/) so you could also check out their [support for Windows](https://docs.substrate.io/v3/getting-started/windows-users/).
```

Comment:
We don't support exactly because Substrate doesn't work on windows. So this is a bit misleading. Not our fault ;)

Reply:
Oh, alright! Yes I didn't notice that. I'll rephrase it.

Reply:
I have rephrased the note. Edit commit added (no rebase/ force-push).

Reply:
Please review and mark as resolved if satisfied.